types/memorystream/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for memorystream 0.3
+// Project: https://github.com/JSBizon/node-memorystream
+// Definitions by: My Self <https://github.com/me>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+/// <reference types="node" />
+
+import { Stream, Duplex } from "stream";
+
+type DataType = string | Buffer | Stream;
+
+interface Options {
+    readable: boolean;
+    writeable: boolean;
+    maxbufsize: number;
+    bufoverflow: number;
+    frequence: number;
+}
+
+declare class MemoryStream extends Duplex {
+    constructor(
+        data?: DataType | DataType[],
```

Comment:
Is this actually potentially an array of strings?

Reply:
yes it is
![image](https://user-images.githubusercontent.com/3430784/51420729-ab70fe00-1bcf-11e9-85d8-184b6848f397.png)